src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+:root {
+  --main-accent: hsl(234, 80%, 12%);
+  --secondary: hsl(244, 8%, 41%);
+  --blue-accent: hsl(193, 100%, 43%);
+}
```

Comment:
What does it mean "secondary"? You need to fix the variable names, they should show their purpose, not their content. 
Also, let's practice SCSS variables instead of CSS.